egs/sprakbanken_swe/s5/local/sprak_data_prep.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+# Copyright 2009-2012  Microsoft Corporation  Johns Hopkins University (Author: Daniel Povey)
+# Copyright 2013-2014  Mirsk Digital Aps (Author: Andreas Kirkedal)
+#Copyright 2016 KTH Royal Institute of Technology (Author: Emelie Kullmann)
+# Apache 2.0.
+
+
+dir=`pwd`/data/local/data
+lmdir=`pwd`/data/local/transcript_lm
+traindir=`pwd`/data/local/trainsrc
+testdir=`pwd`/data/local/testsrc
+rm -rf $lmdir $traindir $testdir $devdir
+mkdir -p $dir $lmdir $traindir $testdir $devdir
+local=`pwd`/local
+utils=`pwd`/utils
+
+
+. ./path.sh
+
+# Checks if python3 is available on the system and install python3 in userspace if not
+# This recipe currently relies on version 3 because python3 uses utf8 as internal 
+# string representation
+
+if ! which python3 >&/dev/null; then
+  echo "Installing python3 since not on your path."
+  pushd $KALDI_ROOT/tools || exit 1;
+  extras/install_python3.sh || exit 1;
+  popd
+fi
+
```

Comment:
I have used a lot from the danish recipe located in ```egs/sprakbanken``` especially for the part of just downloading the scripts from nb.no and extracting txt snd and wav from the downloaded directories, so this part is just a straight copy from the danish script so I'm afraid I can't elaborate it more than that. 

```local/sprak2kaldi.py```and ```local/data_prep.py``` I haven't changed at all so maybe it's a waste putting them up again, I don't know how it would be preferred, to have a copy in ```sprakbanken_swe/local``` or instead just calling the scripts in ```sprakbanken/local``` ? 

I would guess that since both swedish and danish uses a lot of odd characters the encoding would have to be specified. At least that's my experience from working with the swedish txt files on a linux server, it just didn't work unless I explicitly specified it.